### DISCUSSION TITLE
Effect help

### DISCUSSION THREAD

**Discussion by Litrothebronze** [League-of-Fabulous-Developers/FoundryVTT-Fabula-Ultima#113] [Q&A] [answered]
Is there a place with a list of already made examples of effects done? I'm having a lot of trouble getting mine to work. Don't know if I'm miss using the provided Attribute Keys, there is something wrong with my set up, or I'm just an idiot.

**Comment by spyrella** [comment]
There isn't one right now, but I could compile a list in the form of a wiki later today.

**Reply by spyrella** [reply]
I created a spreadsheet where I'll be periodically adding active effects.
https://docs.google.com/spreadsheets/d/1uNgG9rzaOUVfwUasznqKRYiVsMUpmfysoYFkXQWwTyg/edit?usp=sharing

**Reply by belabor** [reply]
thanks, this is really helpful.. I'm guessing there is currently no way to effect suffered damage, i.e. **Defensive Mastery**?

**Reply by spyrella** [reply]
not currently in the system, but someone in the development discord is creating a workaround for it.

**Comment by spyrella** [comment, marked as answer]
In the latest version Defensive Mastery is now doable via the incomingDamage attribute
![image](https://github.com/user-attachments/assets/b7d6d132-8c5e-473d-8c76-63d74862cbed)
In this example you can deduct the damage taken by it's skill level.
